src/widgets/index/index.ts

```diff
@@ -12,12 +13,12 @@ import {
   InitOptions,
   RenderOptions,
   WidgetStateOptions,
+  WidgetSearchParametersOptions,
   ScopedResult,
 } from '../../types';
 import {
   createDocumentationMessageGenerator,
   resolveSearchParameters,
-  enhanceConfiguration,
```

Comment:
it can be removed now, right?

Reply:
Yes but want to do it inside a dedicated PR (only RED!!!!).